custom_functions/mark_evidence.json

```diff
@@ -17,16 +17,16 @@
             "contains_type": [
                 "*"
             ],
-            "description": "Id of object to be added - artifact id, note id, etc.",
+            "description": "The object to marked as evidence. This could be a vault_id, artifact_id, note_id, or if the previous playbook block is an action then \"keyword_argument:results\" can be used with the content_type \"actionrun\".",
```

Comment:
Two things here:
1. "to marked as evidence" should be "to be marked as evidence"
2. container_id is back in, so please add it to the list